### PR TITLE
Attempt to fix PPA building issues

### PR DESCRIPTION
**Pull request by rijnhard**
change all debian deps to python3
drop python2 completely

relates to https://github.com/pwr-Solaar/Solaar/issues/656

Attempting to address https://github.com/pwr-Solaar/Solaar/issues/696

@pfps I dont know who has access to the PPA, it would be useful if we could build a separate branch to test. This is guesswork at this stage

### CODE REVIEW

**Comment by pfps** [comment]
@rijnhard I have been in contact with the person who is apparently responsible for the PPA, but I'm not sure how much they actually understand about how the Solaar PPA actually works.  Do you know how to kick off a test run of the PPA?

**Comment by rijnhard** [comment]
Literally no idea. But I'm starting to think we should scrap the whole
thing and reference skitt's debian Repo instead. It's actively maintained
and he knows what he's doing.

I'll experiment with creating my own ppa when I have time to see how we can
do it and this time around we can add multiple maintainers incase someone
has life getting in the way (look at me taking liberties here).

Are you OK with that?

**Comment by pfps** [comment]
@rijnhard Agreed

**Comment by rijnhard** [comment]
I'm going to keep some notes here as learnings as I figure things out.

1. launchpad is a pain (lol)
2. I'm trying to mirror the solaar repo to launchpad, this is proving helpful https://forum.snapcraft.io/t/maintaining-and-publishing-multiple-to-multiple-channels-via-build-snapcraft-io/12455
3. I want to pull skitts debian packaging and build with that on changes. (TBC)

**Comment by muzena** [comment]
@rijnhard 
Fixed

**Comment by FFY00** [comment]
Closing as discussed in #673.

TLDR: We don't want to keep the packaging directory here but we can create a new repo just for that if there's any interest.
